index.html

```diff
@@ -2595,6 +2595,103 @@ <h3>Data Schemas</h3>
 
       </section>
 
+      <section>
+        <h2>External Resource Integrity</h2>
+        <p>
+          When including a link to an external resource in a VC it is 
+          desireable to know if the resource that is pointed to, is the
+          same at signing time, as at verification time.  This applies to
+          both cases where there is an external resource that is 
+          remotely retrieved, as well as to cases where the issuer and/or
+          verifier may have local cached copies of a resource.
+        </p>
+        <p>
+          It is also desirable to know that the contents of the
+          context(s) utilized in the verifiable credential are the same as
+          used by both the issuer and verifier. 
+        </p>
+        <p>
+          To validate that a resource referenced by a Verifiable Credential is
+          the same at verification time as at issuing time an implementer
+          MAY include a property named <code>resourceIntegrity</code> that
+          stores an array of objects that describe additional integrity
+          metadata about each resource referenced by the VC. If
+          <code>resourceIntegrity</code>
+          is present there MUST be an object in the array for each remote
+          resource.
+        </p>
+        <p>
+          Each object in the
+          <code>resourceIntegrity</code> array MUST contain the following:
+          the URL to the resource named <code>resource</code>, a
+          <code>timestamp</code>
+          that indicates the time at which the hash was computed, the
+          <code>hash</code>
```

Comment:
To be interoperable, it needs to be defined that the `hash` field value is the base64url-encoded representation of the hash value.